content/tutorial/02-advanced-svelte/09-special-elements/06-svelte-body/app-a/src/lib/App.svelte

```diff
@@ -9,7 +9,7 @@
 <!-- creative commons BY-NC http://www.pngall.com/kitten-png/download/7247 -->
 <img
 	class:curious={hereKitty}
-	alt="Kitten wants to know what's going on"
+	alt="Kitten muốn biết chuyện gì đang xảy ra"
```

Comment:
Lỡ dịch rồi chắc mình dịch luôn cả chữ kitten nhỉ?

```suggestion
	alt="Mèo con muốn biết chuyện gì đang xảy ra"
```